src/panels/config/backup/components/ha-backup-summary-card.ts

```diff
@@ -136,20 +140,17 @@ class HaBackupSummaryCard extends LitElement {
       letter-spacing: 0.25px;
       color: var(--secondary-text-color);
       margin: 0;
-      text-overflow: ellipsis;
       overflow: hidden;
-      white-space: nowrap;
```

Comment:
We should keep a fixed height for the banner to avoid UI jump when the text changes and for the data-table scroll.